packages/grafana-ui/src/components/VizRepeater/VizRepeater.tsx

```diff
@@ -172,13 +175,15 @@ export class VizRepeater<V, D = {}> extends PureComponent<Props<V, D>, State<V>>
 
     let resolvedOrientation = this.getOrientation();
 
+    const defaultVizHeight = height / values.length - itemSpacing + itemSpacing / values.length;
```

Comment:
can we add some parentheses to this to make it a bit more clear what operations are happening 😬 that or split out sub expressions into separate named consts 😅